pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/NonPepperKitCreationService.java

```diff
@@ -23,11 +23,17 @@
 @Slf4j
 public class NonPepperKitCreationService {
```

Comment:
Not from this PR but the name of the class NonPepperKitCreationService is interesting.
I would have assumed JuniperKitCreationService

Reply:
That actually was the first naming but we thought this might be used in future for other services since the end point has the potential of getting called from any authorized server. They just need an address and cost objects/settings to have been figured out in DSM